db-connector/src/main/java/uk/nhs/adaptors/connector/model/RequestStatus.java

```diff
@@ -5,8 +5,8 @@
 @Getter
 public enum RequestStatus {
```

Comment:
Maybe we could work on the names:
RequestStatus -> MigrationStatus
RECEIVED -> REQUEST_RECEIVED
EXTRACT_RECEIVED -> EHR_EXTRACT_RECEIVED
COMPLETED -> remove - we will add it when needed